module_mention.lua

```diff
@@ -36,9 +42,10 @@ function Module:OnMessageCreate(message)
 	end
 
 	local mention = false
+	local config = self:GetConfig(message.guild)
 
 	if (message.mentionsEveryone) then
-		mention = true
+		mention = config.ReactOnEveryoneOrHere
```

Comment:
This should be checked in the if, so it fallbacks on the mentionedUsers loop if disabled (that way, it handles messages containing both @everyone and @NotaBot)

Reply:
Done